packages/datastore-extension/src/todo.tsx

```diff
@@ -55,28 +55,28 @@ class TODO extends React.Component<TODOProps, TODOState> {
         <h1>TODO</h1>
         <ol>
           <UseSignal signal={this.state.tableManager.changed}>
-            {() => {
-              console.log(toArray(this.state.tableManager.table.iter()));
-              return toArray(this.state.tableManager.table.iter()).map(row => {
-                console.log(row);
-                return row.show ? (
+            {() =>
+              toArray(this.state.tableManager.table.iter()).map(row =>
```

Comment:
For reference: `@phosphor/algorithm` has a `map` function that could replace `toArray`/`iter()`/`.map()` triplet.